driver.go

```diff
@@ -184,10 +183,11 @@ func (d *DockerOnTop) Mount(request *volume.MountRequest) (*volume.MountResponse
 	}
 	defer activemountsdir.Close() // There is nothing I could do about the error (logging is performed inside `Close()` anyway)
 
-	_, readDirErr := activemountsdir.ReadDir(1) // Check if there are any files inside activemounts dir
-	if errors.Is(readDirErr, io.EOF) {
-		// No files => no other containers are using the volume. Need to mount the overlay
-
+	doMountFs, err := d.activateVolume(request.Name, request.ID, activemountsdir)
+	if err != nil {
+		log.Errorf("Error while activating the filesystem mount: %w", err)
```

Comment:
`%w` is a feature of `fmt.Errorf` only, when you want to create an error _object_. As `log.Errorf` just prints something, rather than creating an object, it uses `fmt.Sprintf` under the hood, which will either silently interpret `%w` as `%v`, or will display some warning or something. In any case,
```suggestion
		log.Errorf("Error while activating the filesystem mount: %v", err)
```

Reply:
Done this one and another one in line 250.